### PR TITLE
Add ru_RU translation

### PR DESCRIPTION
**Pull request by ppv1979**

### CODE REVIEW

File: locale/ru_RU/admin.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  * Copyright (c) 2000-2012 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the ru_RU (Русский) locale.
+  * Translated by Pavel Pisklakov. 
+  *
+  * $Id$
+  -->
+
+<locale name="ru_RU" full_name = "Russian">
```
Comment (mtub) [inline locale/ru_RU/admin.xml]: Could you adjust the name to "Русский" in all files?

File: locale/ru_RU/admin.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+<!DOCTYPE locale SYSTEM "../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * ocs2-devel/locale/ru_RU/admin.xml
+  *
+  * Copyright (c) 2000-2012 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the ru_RU (Русский) locale.
+  * Translated by Pavel Pisklakov.
```
Comment (mtub) [inline locale/ru_RU/admin.xml]: What do you think about putting OCS translation information here: https://pkp.sfu.ca/wiki/index.php?title=Translating_OxS#OCS_Languages (new entry for ru_RU locale, putting your name as a translator in that file)? We could then replace the credit in the XML file with a link to the wiki (see e.g. the ca_ES or de_DE locale files of OMP).

**Comment by mtub** [comment]
Pavel, this looks very good, thanks a lot! Two suggestions, see above. I could do both updates if needed, but wanted to discuss with you first.

File: locale/ru_RU/admin.xml (hunk 2 above)
Comment (ppv1979) [inline locale/ru_RU/admin.xml]: @mtub, yes, you can do it. I think it worth to do in all languages for all OxS projects. Don't you think of making such changes in the whole bunch of translations to all locales?

File: locale/ru_RU/admin.xml (hunk 1 above)
Comment (ppv1979) [inline locale/ru_RU/admin.xml]: @mtub, done.

File: locale/ru_RU/admin.xml (hunk 2 above)
Comment (mtub) [inline locale/ru_RU/admin.xml]: Yes, in the end this would be a good thing to do. Right now I am updating when I encounter these issues...

**Pull request merged**